packages/asset-swapper/src/swap_quoter.ts

```diff
@@ -93,7 +93,7 @@ export class SwapQuoter {
             rfqt,
             tokenAdjacencyGraph,
             liquidityProviderRegistry,
-        } = _.merge({}, constants.DEFAULT_SWAP_QUOTER_OPTS, options);
+        } = { ...constants.DEFAULT_SWAP_QUOTER_OPTS, ...options };
```

Comment:
The `merge()` meant tokens that didn't exist on a network were getting included in the adjacency graph.